core/api/v1/keytransparency.proto

```diff
@@ -338,6 +338,15 @@ message MapRoot {
   repeated bytes log_inclusion = 2;
 }
 
+// LogRootRequest contains the information needed to requetst and verify LogRoot.
+message LogRootRequest {
+  // root_hash is the log root hash.
```

Comment:
optional: I'd avoid using the field name in the comments, because the Go generated code uses a different name. Passim.

Reply:
See above